proto/api/v1/role_service.proto

```diff
@@ -68,6 +84,17 @@ message CreateRoleRequest {
     storage.Role   role = 2;
 }
 
+message GetClustersForPermissionsRequest {
+    Pagination      pagination  = 1;
```

Comment:
This doesn't seem to be used at all currently within the API, am I missing something?
I would have expected something along the lines of:
`paginated.FillPagination(query, req.GetPagination(), X)` within the service.